tests/foreman/ui/test_myaccount.py

```diff
@@ -199,19 +175,11 @@ def test_positive_update_password(self):
         """
         for password in _valid_string_data(max_len=254):
             with self.subTest(password):
-                with self.logged_test_user():
-                    self.my_account.navigate_to_entity()
-                    self.my_account.wait_until_element(
-                        locators['users.password'])
-                    self.my_account.assign_value(
-                        locators['users.password'], password)
-                    self.my_account.assign_value(
-                        locators['users.password_confirmation'], password)
-                    self.my_account.click(common_locators['submit'])
-                    self.my_account.wait_until_element_is_not_visible(
-                        common_locators["notif.success"])
-                    self.login.logout()
-                    self.login.login(self.account_user.login, password)
+                with self.session_test_user() as session:
```

Comment:
so why we create a new user for test_positive_update_language, so test_positive_update_language test has to refactored to not create a new user